cartridge/roles/metrics.lua

```diff
@@ -240,6 +238,17 @@ local function stop()
     metrics_vars.custom_labels = {}
 end
 
+local function set_health_handler(new_handler)
+    handlers['health'] = new_handler or health.default_is_healthy_handler
```

Comment:
It is called `health_handler` in `set_health_handler` and `is_healthy_handler` in `default_is_healthy_handler`. It's a bit inconsistent.